src/main/resources/google_checks.xml

```diff
@@ -95,6 +95,12 @@
              value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT,
                     INSTANCE_INIT, ANNOTATION_DEF, ENUM_DEF"/>
         </module>
+        <module name="SuppressionXpathSingleFilter">
+          <!-- suppresion is required till https://github.com/checkstyle/checkstyle/issues/7541 -->
+          <property name="id" value="RightCurlyAlone"/>
+          <property name="query" value="//RCURLY[parent::SLIST[count(./*)=1]
+                                                 or parent::OBJBLOCK[count(./*)=2]]"/>
```

Comment:
Shouldn't we be able to add an IT test to show it works?

Reply:
Too much effort, to make this hack supported by UTs, it is temporal fix till new Check is implemented.
It just ease problem.

Reply:
Why won't UTs support this now? Is it because we only 1 run check at a time instead of the full config?

Reply:
Yes, extra coding is required and try to avoid this in this workaround fix

Reply:
@romani Does it make sense we trim down the IT run to just the specific check and not run the entire config?
One issue with the entire config is it may produce similar violations from different checks, but isn't that better?

Reply:
in IT we run specific Check now:
https://github.com/checkstyle/checkstyle/blob/1a2d4c2bfbd0e9b86e51d5215f20fbecc4c20778/src/it/java/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/RightCurlyTest.java#L108-L115

So, as we become to use single-filters more and more, we need to extend our ITs to load few modules from config files and execute such few modules(check(s) + filters) on file.
I do not want to do this in this PR, as it will extend scope of fix a lot and delay functional fix. It is better to do this as separate issue.

> One issue with the entire config is it may produce similar violations from different checks. IMO, that might be better to show.

you might talk about CI verification item that run whole config over file, 
for this particular CI, I need Input that have no violations. So I used newly created file as some "prove" that suppression actually works.